Makefile

```diff
@@ -1,5 +1,23 @@
 # We assume an active virtualenv for development
-install:
+include requirements.txt
+VENV_NAME?=.venv
```

Comment:
nit picking: Makefiles are notorious for being difficult to read. I suggest adding a space left and right of `?=` purely for readability.